cmd/launcher/main.go

```diff
@@ -135,6 +135,8 @@ func runSubcommands() error {
 		run = runInteractive
 	case "desktop":
 		run = runDesktop
+	case "uninstall":
+		run = runUninstall
```

Comment:
regarding the windows questions, for now I would just return "not supported" or something that that for now if we're on windows